ivadomed/utils.py

```diff
@@ -34,7 +34,7 @@ def __str__(self):
 
 
 def get_task(model_name):
-    return "classification" if model_name in CLASSIFIER_LIST else "segmentation"
+    return "classification" if model_name in CLASSIFIER_LIST else"segmentation"
```

Comment:
Please revert this change. There should be no change here.

Reply:
fixed.